iidm/iidm-serde/src/test/java/com/powsybl/iidm/serde/EncodingTest.java

```diff
@@ -13,34 +13,43 @@
 
 import java.io.ByteArrayInputStream;
 import java.io.ByteArrayOutputStream;
-import java.io.IOException;
 import java.io.InputStream;
 import java.nio.charset.StandardCharsets;
 
+import static org.junit.jupiter.api.Assertions.fail;
+
 /**
  * @author imagrid team {@literal <rte-mco-imagrid at rte-france.com>}
  */
 class EncodingTest {
 
     @Test
-    void testEncodingISO88591ToISO88591() throws IOException {
-        Network network = NetworkSerDe.read(getClass().getResourceAsStream("/encoding/network.xml"));
-        ByteArrayOutputStream baos = new ByteArrayOutputStream();
-        ExportOptions options = new ExportOptions();
-        options.setCharset(StandardCharsets.ISO_8859_1);
-        NetworkSerDe.write(network, options, baos);
-        InputStream is = new ByteArrayInputStream(baos.toByteArray());
-        ComparisonUtils.compareXml(getClass().getResourceAsStream("/encoding/network.xml"), is);
+    void testEncodingISO88591ToISO88591() {
+        try {
+            Network network = NetworkSerDe.read(getClass().getResourceAsStream("/encoding/network.xml"));
+            ByteArrayOutputStream baos = new ByteArrayOutputStream();
+            ExportOptions options = new ExportOptions();
+            options.setCharset(StandardCharsets.ISO_8859_1);
+            NetworkSerDe.write(network, options, baos);
+            InputStream is = new ByteArrayInputStream(baos.toByteArray());
+            ComparisonUtils.compareXml(getClass().getResourceAsStream("/encoding/network.xml"), is);
```

Comment:
Same principle as for `ComparisonUtils.compareTxt`. The method could be renamed to avoid adding a `fail()`.